homeassistant/components/wolflink/__init__.py

```diff
@@ -105,9 +126,14 @@ async def fetch_parameters(client: WolfClient, gateway_id: int, device_id: int):
 
     By default Reglertyp entity is removed because API will not provide value for this parameter.
     """
+    fetched_parameters = await client.fetch_parameters(gateway_id, device_id)
+    return [param for param in fetched_parameters if param.name != "Reglertyp"]
+
+
+async def fetch_parameters_init(client: WolfClient, gateway_id: int, device_id: int):
```

Comment:
We should probably use this function in `async_setup_entry`.

Reply:
Yes, I guess we should.